docs/UserGuide.adoc

```diff
@@ -602,6 +648,20 @@ Goes to the Offline tab which shows the offline copy of bookmarks
  Different user displays. There are three tabs: dashboard, online view
 (web browser), and offline view (caches and notes).
 
+
+[#imgId-dashboard]
+.Mark's Dashboard Tab View
+image::ui-screenshots/uimockup1_dashboard.png[@Dashboard Tab View, 600, 400]
+
+[#imgId-online]
+.Mark's Online Tab View
+image::ui-screenshots/uimockup1_online.png[@Online Tab View, 600, 400]
+
+[#imgId-offline]
+.Mark's Offline Tab View
+image::ui-screenshots/uimockup1_offline.png[@Offline Tab View, 600, 400]
```

Comment:
@kxyoke Possible to remove the highlighted portion in the mockup (*"4 We need to think about whether to store as a html ..."*) when you do the final user guide draft? If it's too troublesome, we can leave it for now.

Reply:
Will need the highlight there to show that notes have highlight though

Or i change the note content?

Reply:
Ah okay I think I misread it. Not critical to change note content then. 

But can we make the paragraph numbers more prominent? I had the impression that they would be beside the paragraphs (instead of being part of the paragraphs). 

Also, what's `insert list of commands after help calls`?

Reply:
For the paragraph number section, what is the 'they' beside paragraphs that you are referring to? (Does 'they' refer to the commands and/or descriptions?)

Also where can I find the `insert list of commands after help calls`?

Reply:
Sorry, I meant 'they' as in paragraph numbers. 😅 Something like [this](https://github.com/dorcastan/main/blob/temp/docs/images/SuggestedUiOfflineParagraphs.png) (left side)?

I saw `insert list of commands after help calls` in the sidebar of the offline tab, where the bookmark list normally is. Should we keep that area to bookmarks/tags only and display response to the help command in the command box (where Mark's other responses are displayed)?